pkg/kev/converter/kubernetes/project_service.go

```diff
@@ -560,7 +560,7 @@ func (p *ProjectService) healthcheck() (*v1.Probe, error) {
 		initialDelaySeconds := p.livenessProbeInitialDelay()
 		failureThreshold := p.livenessProbeRetries()
 
-		if len(command) == 0 || timoutSeconds == 0 || periodSeconds == 0 ||
+		if len(command) == 0 || len(command[0]) == 0 || timoutSeconds == 0 || periodSeconds == 0 ||
```

Comment:
Just curious: why is `len(command[0]) == 0 ` required?

Reply:
Command is mapped to a string slice, so we need to ensure that at least first element is not empty, so in cases you pass in an empty string as your command validation should kick in preventing you from setting invalid probe.